src/main.rs

```diff
@@ -53,7 +55,7 @@ Save types:
 
 Options:
   -d --datatype=<d>     Define in which datatype to save the result [default: f64].
-  -t --nb_thread=<t>    Use <t> cores to compute the math operation [default: 1].
+  -t --num-thread=<t>   Use <t> cores to compute the math operation [default: 1].
```

Comment:
I thought it was a bad idea to use a dash in a flag name. If you tested and you say that it works, then, ok, it works.

Reply:
I change it to `threads` because of the Args struct it has a strange behavior sur `_` in parameters.